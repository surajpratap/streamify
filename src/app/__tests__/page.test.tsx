import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import Root from "@/app/page";
import Reports from "@/app/reports/page";

describe('Page', () => {
  it('renders a heading', () => {
    render(<Root />)

    const heading = screen.getByRole('heading', { level: 1 })

    expect(heading).toBeInTheDocument()
  })

  it('renders homepage unchanged', () => {
    const { container } = render(<Root />)
    expect(container).toMatchSnapshot()
  })
})
