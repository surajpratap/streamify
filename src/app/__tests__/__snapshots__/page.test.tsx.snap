// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders homepage unchanged 1`] = `
<div>
  <div>
    <h1
      class="text-3xl font-semibold text-white"
    >
      Dashboard
    </h1>
    <div
      class="grid lg:grid-cols-4 md:grid-cols-3 gap-4 mt-4"
    >
      <div
        class="flex rounded-lg border border-gray-200 bg-white shadow-md dark:border-gray-700 flex-col max-w-sm dark:bg-gray-900"
        data-testid="flowbite-card"
      >
        <div
          class="flex h-full flex-col justify-center gap-4 p-6"
        >
          <h5
            class="text-2xl font-bold tracking-tight text-gray-900 dark:text-white"
          >
            Total Users
          </h5>
          <p
            class="font-normal text-gray-700 dark:text-gray-400"
          >
            98976
          </p>
        </div>
      </div>
      <div
        class="flex rounded-lg border border-gray-200 bg-white shadow-md dark:border-gray-700 flex-col max-w-sm dark:bg-gray-900"
        data-testid="flowbite-card"
      >
        <div
          class="flex h-full flex-col justify-center gap-4 p-6"
        >
          <h5
            class="text-2xl font-bold tracking-tight text-gray-900 dark:text-white"
          >
            Active Users
          </h5>
          <p
            class="font-normal text-gray-700 dark:text-gray-400"
          >
            9090
          </p>
        </div>
      </div>
      <div
        class="flex rounded-lg border border-gray-200 bg-white shadow-md dark:border-gray-700 flex-col max-w-sm dark:bg-gray-900"
        data-testid="flowbite-card"
      >
        <div
          class="flex h-full flex-col justify-center gap-4 p-6"
        >
          <h5
            class="text-2xl font-bold tracking-tight text-gray-900 dark:text-white"
          >
            Total Streams
          </h5>
          <p
            class="font-normal text-gray-700 dark:text-gray-400"
          >
            989
          </p>
        </div>
      </div>
      <div
        class="flex rounded-lg border border-gray-200 bg-white shadow-md dark:border-gray-700 flex-col max-w-sm dark:bg-gray-900"
        data-testid="flowbite-card"
      >
        <div
          class="flex h-full flex-col justify-center gap-4 p-6"
        >
          <h5
            class="text-2xl font-bold tracking-tight text-gray-900 dark:text-white"
          >
            Revenue
          </h5>
          <p
            class="font-normal text-gray-700 dark:text-gray-400"
          >
            $ 987,010
          </p>
        </div>
      </div>
      <div
        class="flex rounded-lg border border-gray-200 bg-white shadow-md dark:border-gray-700 flex-col max-w-sm dark:bg-gray-900"
        data-testid="flowbite-card"
      >
        <div
          class="flex h-full flex-col justify-center gap-4 p-6"
        >
          <h5
            class="text-2xl font-bold tracking-tight text-gray-900 dark:text-white"
          >
            Top Artist
          </h5>
          <p
            class="font-normal text-gray-700 dark:text-gray-400"
          >
            Jay Z
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
